### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by davidthesavage**
Depends on: https://github.com/lonelyplanet/rizzo/pull/1302

* Mobile version is on browser sizes < 945px

## Mobile Version
* Navigation collapses to full lines
* Navigation headings in top part become accordion-style expanding sections
* Logo, social media links, and copyright move below navigation

## Desktop Version
* Navigation is broken into 3 columns with headings
* Sub navigation below each 3 column heading
* Further navigation below the nav / subnav reiteration

### CODE REVIEW

File: ui/components/footer/_footer.html.hbs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<section class="footer">
+  <div class="footer-wrapper">
```
Comment (jcreamer898) [inline ui/components/footer/_footer.html.hbs]: footer__wrapper?

File: ui/components/footer/_footer.html.hbs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              <a href="{{url}}">{{title}}</a>
+            </li>
+          {{/each}}
+        </ul>
+      </div>
+    </section>
+    <section class="footer__bottom">
+      <div class="footer__bottom__middle">
+        <div class="footer__bottom__middle__logo">
+          <a class="footer-logo icon--lp-logo icon--footer-logo" href="http://www.lonelyplanet.com" title="Return to homepage"></a>
```
Comment (jcreamer898) [inline ui/components/footer/_footer.html.hbs]: Was this logo at all similar to the one we had in the header?

I could envision a scenario in which we have one `.logo` class and you have modifiers for its position...

```scss
.logo {} 
.logo--bottom {}
.logo--white {}

// etc
```

File: ui/components/footer/_footer.html.hbs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            {{/each}}
+          </ul>
+        </nav>
+      </div>
+      <div class="footer__bottom__logo">
+        <a class="footer-logo icon--lp-logo icon--footer-logo" href="http://www.lonelyplanet.com" title="Return to homepage"></a>
+      </div>
+      {{#with footer_social_media_links}}
+        <div class="footer__bottom__social-media">
+          <a href="{{facebook}}" class="footer__bottom__social-media__link">
```
Comment (jcreamer898) [inline ui/components/footer/_footer.html.hbs]: Similar to the above question, and maybe so or maybe not here, but is this a "reusable" thing?

File: ui/components/footer/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+  }
+  .accordion-target {
+    position: relative;
+    bottom: 14px;
+    float: right;
+    width: 50%;
+    height: 50px;
+    cursor: pointer;
+  }
+  .icon--chevron-down {
```
Comment (jcreamer898) [inline ui/components/footer/_footer.scss]: Seems like this kind of class should potentially live somewhere common?

File: ui/components/navigation/_navigation.html.hbs [hunk 5]
```diff
@@ -8,7 +8,7 @@
     {{> "components/search/search" }}
     <div class="nav-container__inner">
       <ul class="navigation__menu">
-       {{#each navigation}}
+       {{#each navigation.header_items}}
```
Comment (jcreamer898) [inline ui/components/navigation/_navigation.html.hbs]: I think I like it the old way a bit better, it's nice to not HAVE to dig down deeper into the data hierarchy. Can we try to keep it just as `{{#each navigation}}`, or `{{#each header_items}}`?

File: ui/components/footer/_footer.scss (hunk 4 above)
Comment (JoeShep) [inline ui/components/footer/_footer.scss]: Do you mean `.icon--chevron-down` or `.accordion-target`?

Comment (jcreamer898) [inline ui/components/footer/_footer.scss]: chevron

File: ui/components/footer/_footer.html.hbs (hunk 3 above)
Comment (JoeShep) [inline ui/components/footer/_footer.html.hbs]: Explain more, please

File: ui/components/footer/_footer.scss (hunk 4 above)
Comment (JoeShep) [inline ui/components/footer/_footer.scss]: I does live in Rizzo, just to declare a background image. This styling is over and above that.

**Pull request merged**